src/core/WorkflowsClientProfile/OpportunityPreview.tsx

```diff
@@ -72,47 +79,48 @@ const FormPreview = styled.img<Pick<StatusPalette, "border">>`
   width: ${rem(205)};
 `;
 
-export const CompliantReportingPreview = observer(
-  ({ client }: ClientProfileProps) => {
-    if (!client.opportunities.compliantReporting) return null;
+export const OpportunityPreview = observer(({ client }: ClientProfileProps) => {
+  // We will choose the first opportunity for now,
+  // until we have a design for how to preview multiple opportunities per client
+  const opportunity = values(client.opportunities).filter(
+    (opp) => opp !== undefined
+  )[0];
```

Comment:
If you just want the first match, you can user `arr.find()` instead of `arr.filter()[0]`. This has the advantage of both being more terse and faster (on average).

```suggestion
  const opportunity = values(client.opportunities).find(
    (opp) => opp !== undefined
  );
```